modules/react-native-status/android/build.gradle

```diff
@@ -20,19 +20,8 @@ dependencies {
     implementation 'com.facebook.react:react-native:+' // from node_modules
     compile 'com.github.status-im:function:0.0.1'
 
-    // WARNING: If you change this, make sure the GitHub release of the .aar exists.
-    // WARNING: status-go is downloaded in Makefile and has a hardcoded version too.
     String statusGoVersion = getStatusGoVersion()
     final String statusGoGroup = 'status-im', statusGoName = 'status-go'
```

Comment:
there is no need for these to be variables anymore, they will be more readable inside `implementation` call later